app/model/Event/Education.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Model\Event;
+
+use Cake\Chronos\Date;
+use Model\Common\UnitId;
+use Model\Skautis\ISkautisEvent;
+use Nette\NotImplementedException;
+use Nette\SmartObject;
+
+/**
+ * @property-read SkautisCampId $id
+ * @property-read string $displayName
+ * @property-read Date $startDate
+ * @property-read Date $endDate
+ * @property-read string $state
+ */
+class Education implements ISkautisEvent
```

Comment:
Místo PHPDoců prosím použít typed properties pro všechny properties v této třídě (už nechci víc zahlcovat X komentáři :slightly_smiling_face:). Pak to nějak hromadně zmigruju ve starým kódu.

Reply:
upraveno